project/Component/NefModels/CarbonModel.swift

```diff
@@ -2,25 +2,54 @@
 
 import Foundation
 
+/// Represents the configuration for rendering carbon snippets.
 public struct CarbonModel: Codable, Equatable {
+    /// Code to render into Carbon image.
     public let code: String
+    
+    /// Style to apply to the generated snippet.
     public let style: CarbonStyle
     
+    /// Initializes a `CarbonModel`
+    ///
+    /// - Parameters:
+    ///   - code: code to render into Carbon image.
+    ///   - style: style to apply to the generated snippet.
     public init(code: String, style: CarbonStyle) {
         self.code = code
         self.style = style
     }
 }
 
-// MARK: - Style
+/// Describes the style to apply to Carbon snippet.
 public struct CarbonStyle: Codable, Equatable {
+    /// Background color
     public let background: Color
+    
+    /// File size.
```

Comment:
I think *file size* is misguiding.
```suggestion
    /// Image size.
```